### ISSUE TITLE
JavaScript error

### ISSUE THREAD

**Issue by GleepZorp** [riverrrrrr/discord-soundcloud#9]
I used the Bug-Free 3.0.0 release of the installer and the application works, but the Discord functionality does not. 

I got this error at some point as well:
![capture2](https://user-images.githubusercontent.com/35981883/52921386-09caf280-32dc-11e9-9d02-5747fe77ffd1.PNG)

**Comment by treypeng** [comment]
Yep getting the same, and also ... it just doesn't work on macOS High Sierra at all.

**Comment by ChineseMarc** [comment]
@treypeng #7 should fix that (works on linux mint)
